Cargo.toml

```diff
@@ -165,7 +165,10 @@ bench = false
 # To use a development version of a dependency please use a global override here
 # changing versions in each sub-crate of the workspace is tedious
 [patch.crates-io]
-# reedline = { git = "https://github.com/nushell/reedline.git", branch = "main"}
+reedline = { git = "https://github.com/nushell/reedline.git", branch = "main" }
+dtparse = { git = "https://github.com/nibon7/dtparse.git", branch = "chrono" }
```

Comment:
These type of things are fine for now, but we can't publish releases unless the dependency crates are in crates.io.

Reply:
We're waiting for the upstream to merge related PRs and this is temporary to make to the audit CI works :)

Reply:
ok, i put checkboxes in the description so we can follow that status easier. Thanks!